HTML/packages.html

```diff
@@ -0,0 +1,79 @@
+        <h2>Our Packages</h2>
+        
+        <div class="package" id="ella-one-day">
+            <h3>Ella (One Day)</h3>
+            <img src="ella.jpg" alt="Ella">
+            <p>Explore the scenic beauty of Ella in just one day. Perfect for families, couples, and groups. Accommodation in three-star hotels included.</p>
+            <a href="book.html?package=ella-one-day">Book Now</a>
+        </div>
+        
+        <div class="package" id="kandy-one-day">
+            <h3>Kandy (One Day)</h3>
+            <img src="kandy.jpg" alt="Kandy">
+            <p>Visit the historic city of Kandy and its cultural landmarks in one day. Ideal for families, couples, and groups. Accommodation in three-star hotels included.</p>
+            <a href="book.html?package=kandy-one-day">Book Now</a>
+        </div>
+        
+        <div class="package" id="yala-one-day">
+            <h3>Yala National Park (One Day)</h3>
+            <img src="yala.jpg" alt="Yala National Park">
+            <p>Experience a thrilling safari at Yala National Park. Suitable for families, couples, and groups. Accommodation in three-star hotels included.</p>
+            <a href="book.html?package=yala-one-day">Book Now</a>
+        </div>
+        
+        <div class="package" id="aluthgama-one-day">
+            <h3>Aluthgama Fishing Tour (One Day)</h3>
+            <img src="aluthgama.jpg" alt="Aluthgama Fishing Tour">
+            <p>Enjoy a day of fishing in Aluthgama. Great for families, couples, and groups. Accommodation in three-star hotels included.</p>
+            <a href="book.html?package=aluthgama-one-day">Book Now</a>
+        </div>
+        
+        <div class="package" id="madu-one-day">
+            <h3>Madu River Boat Safari (One Day)</h3>
+            <img src="madu.jpg" alt="Madu River Boat Safari">
+            <p>Take a serene boat safari on the Madu River. Perfect for families, couples, and groups. Accommodation in three-star hotels included.</p>
+            <a href="book.html?package=madu-one-day">Book Now</a>
+        </div>
+        
+        <div class="package" id="mirissa-one-day">
+            <h3>Mirissa (One Day)</h3>
+            <img src="mirissa.jpg" alt="Mirissa">
+            <p>Relax on the beautiful beaches of Mirissa. Ideal for families, couples, and groups. Accommodation in three-star hotels included.</p>
+            <a href="book.html?package=mirissa-one-day">Book Now</a>
+        </div>
+        
+        <div class="package" id="2-day-package">
+            <h3>2 Day Package: Ella, Nuwara Eliya, Kandy, Sigiriya, Matale</h3>
+            <img src="2-day-tour.jpg" alt="2 Day Tour">
+            <p>Experience the highlights of Ella, Nuwara Eliya, Kandy, Sigiriya, and Matale in a 2-day tour. Ideal for families, couples, and groups. Accommodation in three-star hotels included.</p>
+            <a href="book.html?package=2-day-package">Book Now</a>
+        </div>
+        
+    </section>
+    
+    <footer>
```

Comment:
We should have the same footer to make website consistent. Copy from home code. (link the css files)